### DISCUSSION TITLE
heating on and off

### DISCUSSION THREAD

**Discussion by Dj-spooky** [Mariusthvdb/custom-ui#69] [Q&A] [answered]
hey,
(Dutch below)
I would like when my boiler starts to burn that the icon becomes a flame,
and when the central heating boiler is turned off, there is a flame with a line through it,
the circuit just goes with 'on' and 'off'
see image below.
I just want it in an entity card
can you help me? Thanks in advance!
------------------------------------------------
Dutch

hey, 
ik wil graag als me cv ketel gaat branden dat het icoon een flammetje word,
en als de cv ketel uit staat dat er dan een flammetje komt met een streep erdoorheen,
de schakeling gaat gewoon met 'on' en 'off '
zie afbeeling beneden.
ik wil het gewoon in een entity kaart hebben
kan je me helpen? bedankt alvast!

![Schermafbeelding 2022-11-20 014013](https://user-images.githubusercontent.com/109301472/202877335-a71d85cf-cd50-45d6-90b1-07f6a5415f9c.png)
![Schermafbeelding 2022-11-20 014238](https://user-images.githubusercontent.com/109301472/202877394-a37bd1e1-d5a8-4d86-aa60-09874e9c67f4.png)

**Comment by Mariusthvdb** [comment, marked as answer]
Hi,

Not using mushroom so not sure if custom-Ui works on that. It does Not work on a core Entity card. You need card-mod for that.

Entities card is no issue at all.
Just follow the examples.

Cheers.
